Rubberduck.Parsing/Symbols/FunctionDeclaration.cs

```diff
@@ -67,23 +61,15 @@ public sealed class FunctionDeclaration : Declaration, IParameterizedDeclaration
                 null,
                 attributes)
         {
-            _parameters =
-                member.Parameters.Select(decl => new ParameterDeclaration(decl, this, module))
-                    .ToList();
+            AddParameters(member.Parameters.Select(decl => new ParameterDeclaration(decl, this, module)));
         }
 
-        public IEnumerable<ParameterDeclaration> Parameters => _parameters.ToList();
-
-        public void AddParameter(ParameterDeclaration parameter)
+        protected override bool Implements(ICanBeInterfaceMember interfaceMember)
         {
-            _parameters.Add(parameter);
+            return interfaceMember.DeclarationType == DeclarationType.Function
+                && interfaceMember.IsInterfaceMember
+                && IsInterfaceImplementation
+                && IdentifierName.Equals($"{interfaceMember.InterfaceDeclaration.IdentifierName}_{interfaceMember.IdentifierName}");
```

Comment:
How about extracting the string into a function `InterfaceMemberImplementationName` or something like that to make it a bit more self-documenting?

Reply:
Works for me. Inbound.